Minebot/src/main/java/net/famzangl/minecraft/minebot/ai/path/PlantPathFinder.java

```diff
@@ -198,7 +203,11 @@ protected void notFacingBlock(AIHelper aiHelper) {
 	private final PlantType type;
 
 	public PlantPathFinder(PlantType type) {
-		this.type = type;
+		if (type != null) {
```

Comment:
… then you can set this to non-null.

In newer code, I often do a Objects.requireNonNull(type, "type") to catch errors early.

Reply:
I see! I can change it with the next push I suppose :) I still don't know how to use airbridge and fill though